lib/helpers.js

```diff
@@ -26,6 +26,17 @@ _beforeEach.withApp = function(app) {
   });
 }
 
+_beforeEach.cleanDatasource = function(dsName) {
+  beforeEach(function() {
+    if(!dsName) dsName = 'db';
+
+    if (this.app) {
+      this.app.datasources[dsName].automigrate();
```

Comment:
@jrschumacher do you have time to fix this? If not I can create a pull request and fix this in order to land this.

You should be able to just add a callback to `beforeEach`.